shared/src/main/kotlin/com/egm/stellio/shared/model/CompactedEntity.kt

```diff
@@ -42,6 +46,20 @@ typealias CompactedEntity = Map<String, Any>
 typealias CompactedAttributeInstance = Map<String, Any>
 typealias CompactedAttributeInstances = List<CompactedAttributeInstance>
 
+val JSONLD_COMPACTED_ATTRIBUTE_CORE_MEMBERS =
```

Comment:
`JSONLD_LANGUAGEMAP_TERM` should be in the list (if it is not in the list, it is a trap for future users of your list)